frontend-react/src/components/header/SignInOrUser.tsx

```diff
@@ -24,23 +24,24 @@ const SignInOrUser = () => {
         <div className="prime-user-account">
             <span id="emailUser">{user}</span>
             <br />
-            <a
-                href="/"
+            {/* Okta handles our logoutRedirectUri, this should _not_ be an anchor tag! */}
+            <button
                 id="logout"
                 onClick={() => {
                     logout(oktaAuth);
                 }}
-                className="usa-link"
+                className="usa-button usa-button--unstyled"
             >
                 Log out
-            </a>
+            </button>
         </div>
     ) : (
-        <NavLink to="/login">
+        <USLink href="/login">
```

Comment:
I realize this existed beforehand so it's out of the scope of this changeset, but it's invalid html to have a `<button>` inside an `<a>`.  Could we turn that into a navigation in an `onClick`?

Reply:
Yep! I believe we have a few of these in here that were quick jobs in the refactor that need to be revisited. I don't mind going back and just nixing them all for an `onClick(() => navigate("..."))` in the button itself. There was also some dodgy CSS overrides I had to write to get styling right because of the bad nesting. Would love to rip it out and never let it see the light of `master`

Reply:
No problem!  As far as I know, modern browsers have become a lot more lenient so nothing like this explicitly breaks the UX.  It's just not great for accessibility since an element that can be actioned shouldn't be nested into _another_ element that can be actioned.

I can add a ticket for this, and we can come back to it later to keep your changeset specific to the replacement of links! :)

Reply:
Feel free to update!  https://app.zenhub.com/workspaces/experience-607d9d5e68b95200150fec37/issues/gh/cdcgov/prime-reportstream/7838

Reply:
Oops! Forgot to resolve this; just reviewed the ticket and it looks good! Just added `needs refinement` tag